v4/source/plugins.yaml

```
@@ -553,3 +553,50 @@
             }
 
             ok, resp = Client.GetMarketplacePlugins(filter)
+  /plugins/first_admin_visit_marketplace:
+    get:
+      tags:
+        - system
+      summary: Retrieves the system-level status that specifies that at least one System Admin has visited the in-product Plugin Marketplace.
```

The summary is quite long and clutters the side bar. Can we shorten it a bit to something like 
```suggestion
      summary: Get if the Plugin Marketplace has been visited`
```